apis/nucleus/src/index.js

```diff
@@ -461,19 +462,31 @@ function nuked(configuration = {}) {
             }
             const onSelectionActivated = () => fieldSels.emit('selectionActivated');
             const onSelectionDeactivated = () => fieldSels.emit('selectionDeactivated');
-            this._instance = ListBoxPortal({
-              element,
-              app,
-              fieldIdentifier,
-              qId,
-              options: getListboxPortalOptions({
-                onSelectionActivated,
-                onSelectionDeactivated,
-                ...options,
-              }),
-              stateName: options.stateName || '$',
+            let renderCount = 0;
+
+            return new Promise((resolve) => {
+              const renderedCallback = () => {
+                renderCount++;
+                if (renderCount === 1) {
+                  resolve();
+                }
```

Comment:
This looks a bit unnecessary?

Reply:
Sure, it can be simplified, I've updated it.